source/scheme/mus_scheme_header_module.f90

```diff
@@ -104,6 +104,8 @@ module mus_scheme_header_module
     character(len=labelLen) :: layout
     !> scheme relaxation type Ex: BGK, MRT, bgk_pl, bgk_cy, bgk_cs...
     character(len=labelLen) :: relaxation
+    !> equilibrium order Ex: first, second
+    character(len=labelLen) :: order
```

Comment:
Should be replaced with relaxation variant. This option is to be introduced in change merge.